src/api/OrdersApi.js

```diff
@@ -104,14 +104,10 @@ export default class OrdersApi {
   placeOrderWithHttpInfo(id, opts) {
     opts = opts || {};
 
-    const _placeOrderRequest = PlaceOrderRequest.constructFromObject(
-      opts,
-      new CreateOrderRequest()
-    );
-    let postBody = _placeOrderRequest;
+    let postBody = opts['placeOrderRequest'];
```

Comment:
Also strange, investigating.

Reply:
Ah, that's the classic case of two 200 examples block overriding parts of the swagger file.